src/covariance.h

```diff
@@ -157,15 +170,40 @@ matrix<T> get_covariance_lower_chol(const vector<T>& theta, int n_visits, std::s
   return result;
 }
 
+// Creates a new spatial covariance cholesky.
+template <class T>
+matrix<T> get_spatial_covariance_lower_chol(const vector<T>& theta, const matrix<T>& distance, std::string cov_type) {
+  matrix<T> result;
+  if (cov_type == "gp_exp") {
```

Comment:
```suggestion
  if (cov_type == "sp_exp") {
```
probably easiest to search for `gp_` in the whole package and replace by `sp_` :-)